### PR TITLE
[Android] Highlight layers button when it is activated

### PR DESCRIPTION
**Pull request by RedAuburn**
https://github.com/user-attachments/assets/b4ca6cde-431f-488e-b25c-d3a2aabe63e8

Closes https://github.com/organicmaps/organicmaps/pull/8821

Workarounds https://github.com/organicmaps/organicmaps/issues/3508

### CODE REVIEW

**Review by Jean-BaptisteC — APPROVED** [review]
✅ Pixel 6 - Android 14

**Review by biodranik — COMMENTED** [review]
Thanks! LGTM with a formatting comment.

File: android/app/src/main/java/app/organicmaps/maplayer/LayersButton.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import android.content.Context;
+import android.util.AttributeSet;
+
+import com.google.android.material.floatingactionbutton.FloatingActionButton;
+
+import app.organicmaps.R;
+
+public class LayersButton extends FloatingActionButton {
+    private boolean mAreLayersActive = false;
```
[the remark this answers is not in the document]
Comment (RedAuburn) [reply android/app/src/main/java/app/organicmaps/maplayer/LayersButton.java]: fixed :)

**Review by biodranik — APPROVED** [review]
Thanks! This is a quick workaround, of course, just to teach users about layers in their current form. A better solution could be:
- Think about moving layers button into the bottom as a 5th button
- Focus on switchable "modes" instead of layers: driving, hiking (shows isolines), cycling (shows isolines), public transport, etc., then we don't need many complex combinations of the currently active layer

Maybe there are some other ideas. CC @euf

**Pull request merged**